templates/applicants.phtml

```diff
@@ -53,7 +53,27 @@
                 </div>
 
             </form>
+            <nav aria-label="Page navigation mt-5">
+                <ul class="pagination justify-content-center">
+                    <li class="page-item <?php if($page <= 1){ echo 'disabled'; } ?>">
+                        <a class="page-link"
+                           href="<?php if($page <= 1){ echo '#'; } else { echo "?page=" . ($page-1); } ?>">Previous</a>
+                    </li>
+
+                    <?php for($i = 1; $i <= $count; $i++ ): ?>
+                        <li class="page-item <?php if($page == $i) {echo 'active'; } ?>">
+                            <a class="page-link" href="/applicants?page=<?= $i; ?>"> <?= $i; ?> </a>
+                        </li>
+                    <?php endfor; ?>
+
+                    <li class="page-item <?php if($page >= $count) { echo 'disabled'; } ?>">
+                        <a class="page-link"
+                           href="<?php if($page >= $count){ echo '#'; } else {echo "?page=". ($page+1); } ?>">Next</a>
+                    </li>
+                </ul>
+            </nav>
```

Comment:
Now that we've added the pagination for applicants, we may well want to add the same to the other tables in the portal. Therefore it would be quite nice if this code were re-usable. 

Would suggest putting the pagination display into a generic pagination view helper that could then be used elsewhere if needed.